code/modules/holodeck/holo_effect.dm

```diff
@@ -109,11 +109,29 @@
 
 /obj/effect/holodeck_effect/mobspawner/penguin
 	mobtype = /mob/living/simple_animal/pet/penguin/emperor
-	
+
 /obj/effect/holodeck_effect/mobspawner/penguin/Initialize()
 	if(prob(1))
 		mobtype = /mob/living/simple_animal/pet/penguin/emperor/shamebrero
 	return ..()
 
 /obj/effect/holodeck_effect/mobspawner/penguin_baby
 	mobtype = /mob/living/simple_animal/pet/penguin/baby
+
+/obj/effect/holodeck_effect/mobspawner/cat
+	mobtype = /mob/living/simple_animal/pet/cat
+
+/obj/effect/holodeck_effect/mobspawner/butterfly
+	mobtype = /mob/living/simple_animal/butterfly
+
+/obj/effect/holodeck_effect/mobspawner/clown
+	mobtype = list (/mob/living/simple_animal/hostile/retaliate/clown = 10,
+	/mob/living/simple_animal/hostile/retaliate/clown/banana = 6, /mob/living/simple_animal/hostile/retaliate/clown/honkling = 6,
+	/mob/living/simple_animal/hostile/retaliate/clown/fleshclown = 3, /mob/living/simple_animal/hostile/retaliate/clown/longface = 3,
+	/mob/living/simple_animal/hostile/retaliate/clown/mutant = 1, /mob/living/simple_animal/hostile/retaliate/clown/mutant/blob = 1)
+
+/obj/effect/holodeck_effect/mobspawner/psycho
+	mobtype = list (/mob/living/simple_animal/hostile/psycho/regular = 9,
```

Comment:
An unrestricted holodeck program is loading in mobs from "TheFactory", an away mission which was removed for being extremely broken?
![image](https://user-images.githubusercontent.com/3241376/122023822-8297c780-cd8d-11eb-9fb5-dabb897b9c34.png)

Reply:
Yea, I chose them cause they're gimmicky and look nice. They don't deal enviro damage and won't do a lot of damage when set loose. Keep in mind this is a restricted program, available with emag only, and is not as dangerous as beephobia or plasma flood.

Reply:
Silicons can enable restricted programs as well…